src/Type/Symfony/EnvelopeReturnTypeExtension.php

```diff
@@ -33,15 +36,15 @@ public function getTypeFromMethodCall(
 	): Type
 	{
 		if (count($methodCall->getArgs()) === 0) {
-			return new ArrayType(new MixedType(), new ArrayType(new MixedType(), new ObjectType('Symfony\Component\Messenger\Stamp\StampInterface')));
+			return new ArrayType(new ClassStringType(), new ArrayType(new IntegerType(), new ObjectType('Symfony\Component\Messenger\Stamp\StampInterface')));
```

Comment:
And here... how do I say that the ClassStringType is actually a `class-string<StampInterface>`?

Reply:
By checking out the table here https://phpstan.org/developing-extensions/type-system or API reference here https://apiref.phpstan.org/1.9.x/PHPStan.Type.Type.html

Reply:
GenericClassStringType... okay, solved